conversationgenome/analytics/WandbLib.py

```diff
@@ -26,53 +26,42 @@
 class WandbLib:
     verbose = False
 
-    def init_wandb(self, data=None):
+    def init_wandb(self, config=None, data=None):
+        if config:
+            #initialize data:
+            wallet = bt.wallet(config=config)
+            subtensor = bt.subtensor(config=config)
+            metagraph = subtensor.metagraph(config.netuid)
+            my_hotkey=wallet.hotkey.ss58_address
+            my_uid = metagraph.hotkeys.index(my_hotkey)
```

Comment:
Some of these look fragile and this in particular was the same point of vulnerability we had to patch